utils/firestore.js

```diff
@@ -184,6 +185,210 @@ const updateParticipantData = async (id, data) => {
+            fieldMapping.dataDestruction.userProfileNameSuffixName,
+            'query',
+            fieldMapping.autogeneratedConsentDate,
+            fieldMapping.participantMap.consentFormSubmitted,
+            fieldMapping.dataDestruction.informedConsentDateSigned,
+            fieldMapping.dataDestruction.informedConsentVersion,
+            fieldMapping.dataDestruction.hipaaAuthorizationDateSigned,
+            fieldMapping.dataDestruction.hipaaAuthorizationFlag,
+            fieldMapping.dataDestruction.hipaaAuthorizationVersion,
+            fieldMapping.dataDestruction.firebaseAuthenticationEmail,
+            fieldMapping.firebaseAuthenticationFirstAndLastName,
+            fieldMapping.authenticationPhone,
+            fieldMapping.signInMechanism,
+            fieldMapping.preferredLanguage,
+            fieldMapping.preferredName,
+            fieldMapping.dataDestruction.birthMonth,
+            fieldMapping.dataDestruction.birthDay,
+            fieldMapping.dataDestruction.birthYear,
+            fieldMapping.dataDestruction.dateOfBirth,
+            fieldMapping.cellPhone,
+            fieldMapping.homePhone,
+            fieldMapping.otherPhone,
+            fieldMapping.prefEmail,
+            fieldMapping.additionalEmail1,
+            fieldMapping.additionalEmail2,
+            fieldMapping.additionalEmail3,
+            fieldMapping.address1,
+            fieldMapping.address2,
+            fieldMapping.city,
+            fieldMapping.state,
+            fieldMapping.zip,
+            fieldMapping.canWeVoicemailMobile,
+            fieldMapping.canWeVoicemailHome,
+            fieldMapping.canWeVoicemailOther,
+            fieldMapping.canWeText,
+            fieldMapping.prefContactMethod,
+            fieldMapping.haveYouEverBeenDiagnosedWithCancer,
+            fieldMapping.whatYearWereYouDiagnosed,
+            fieldMapping.whatTypeOfCancer,
+            fieldMapping.anyCommentsAboutYourCancerDiagnosis,
+            fieldMapping.derivedAge,
+            fieldMapping.dataDestruction.userProfileSubmittedFlag,
+            fieldMapping.autogeneratedProfileSubmittedTime,
+            fieldMapping.participantMap.consentFormSubmitted,
+            fieldMapping.verificationStatus,
+            fieldMapping.autogeneratedSignedInTime,
+            fieldMapping.autogeneratedVerificationStatusUpdatedTime,
+            // These are not in either the field mapping or the concept ID DB lookup, but are generated on the user during the creation process
+            983784715, 
+            700668490,
+            430184574,
+            507120821,
+            383945929
+        ];
+        // Pulling these out specifically instead of putting them in the above array
+        // because they're used in subsequent logic
+        const { token, Connect_ID } = prevUserData;
+        obj = {
+            state: {
+                uid,
+                ...defaultStateFlags
+            },
+            230663853: 353358909,
```

Comment:
does syntax allow for these keys to be in their named constant form to make code reading easier?

Reply:
It does. This is only in this form because it's repeated code from generateToken and getToken, which I was hesitant to mess with as outside of the scope of my work. I'll update this one; should I change generateToken and getToken while I'm at it or leave them be?